tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -1226,7 +1227,7 @@ async def test_nft_set_did(two_wallet_nodes: Any, trusted: Any) -> None:
         await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
 
     # Check DID NFT
-    time_left = 5.0
+    time_left = 10.0
     coins_response = {}
     while time_left > 0:
```

Comment:
can you migrate these tests to new utility functions `make_new_block_with` and `wait_rpc_state_condition`